dpnp/backend/extensions/lapack/gesv_batch.cpp

```diff
@@ -242,9 +376,27 @@ std::pair<sycl::event, sycl::event>
     const std::int64_t nrhs =
         (dependent_vals_nd > 2) ? dependent_vals_shape[1] : 1;
 
-    sycl::event gesv_ev =
-        gesv_batch_fn(exec_q, n, nrhs, batch_size, coeff_matrix_data,
+    auto const &coeff_matrix_strides = coeff_matrix.get_strides_vector();
+    auto const &dependent_vals_strides = dependent_vals.get_strides_vector();
```

Comment:
Why do we need to declared them outside the `ifdef`?

Reply:
Because `...strides` are only needed for `mkl_lapack::getrf/getrs` and is not passed to `mkl_lapack::gesv`

Reply:
Do we need to move these declaration under below `ifdef`, because they looks unused for use case when `if not defined (USE_ONEMKL_INTERFACES)`

Reply:
Oh, I had not noticed that. It should be moved.
Thank you for pointing it out.